app/controllers/Application.scala

```diff
@@ -111,14 +111,13 @@ class Application(
   def newContributionsLanding(countryCode: String): Action[AnyContent] = maybeAuthenticatedAction().async { implicit request =>
     type Attempt[A] = EitherT[Future, String, A]
     implicit val settings: Settings = settingsProvider.settings()
-    val visitToken = VisitToken(new SecureRandom())
```

Comment:
this token was generated server side, but suddenly we remembered about caching.  We can trust the client side more to produce a reliable random number in that case.